queries/thrift/highlights.scm

```diff
@@ -27,15 +27,27 @@
 
 ; Fields
 
-(field) @field
+(field_identifier) @field
 
 ; Parameters
 
-(parameter) @parameter
+(param_identifier) @parameter
 
 ; Variables
 
 (identifier) @variable
+(annotation_identifier) @variable
```

Comment:
Maybe `@attribute`?

Reply:
```thrift
struct A {
    1: required string faststr,
    2: required string string(pilota.rust_type = "string"),
}
```
pilota.rust_type is the annotation_identifier, I'm indifferent but if you think that's better off as an attribute then sure, but I think it's better for you to first see an example

Reply:
Shouldn't that be `@parameter` then?

Reply:
hmm, not really since it's not a function call, it's more "annotating" to the thrift compiler how certain fields should be dealt with in certain languages/scenarios

Reply:
So `@attribute` seems better after all.

Reply:
Thought it over, yes I agree. Gonna make a couple more changes, like fixing how enum_identifier and enum_member were handled, and rename this to attribute